lib/datasource/terraform-module/index.ts

```diff
@@ -132,6 +134,11 @@ export async function getReleases({
     if (pkgUrl.startsWith('https://registry.terraform.io/')) {
       dep.homepage = `https://registry.terraform.io/modules/${repository}`;
     }
+    // set published date for latest release
+    const currentVersionIndex = dep.releases.findIndex((release) => {
+      return res.version === release.version;
+    });
+    dep.releases[currentVersionIndex].releaseTimestamp = res.published_at;
```

Comment:
```suggestion
    const currentVersion = dep.releases.find((release) => {
      return res.version === release.version;
    });
    if (currentVersion) {
      currentVersion.releaseTimestamp = res.published_at;
    }
```
use find and add null check